scipy/stats/tests/test_kdeoth.py

```diff
@@ -19,7 +19,15 @@ def test_kde_1d():
     gkde = stats.gaussian_kde(xn)
 
     # evaluate the density function for the kde for some points
-    xs = np.linspace(-7,7,501)
+    xx = np.asarray([0.1, 0.5, 0.9])
+    loc, scale = gkde.dataset, np.sqrt(gkde.covariance)
+    assert_allclose(
```

Comment:
Just taking the opportunity to add PDF tests?

Reply:
Pretty much yes. Was checking that it works the way I think it does, added it as a test. Can remove if desired.

Reply:
They are probably fine. I'll take a look today.